src/test/java/HomeWork19Test.java

```diff
@@ -27,7 +27,7 @@ public void openPlayList()throws InterruptedException{
             Thread.sleep(3000);
         }
     private void deletePlayList()throws InterruptedException{
-        WebElement deleteButton = driver.findElement(By.cssSelector("button.btn-delete-playlist"));
+        WebElement deleteButton = driver.findElement(By.cssSelector("button.del btn-delete-playlist"));
```

Comment:
The locator is not good

You can find an element by using only css class 
![Koel - Google Chrome](https://user-images.githubusercontent.com/49252454/211909476-8f35ed2c-1655-4b89-a572-91a7331d3c68.jpg)